chacha20/Cargo.toml

```diff
@@ -37,6 +37,7 @@ expose-core = []
 force-soft = []
 hchacha = ["cipher"]
 legacy = ["cipher"]
+nightly = []
```

Comment:
For comparison, the `aes` and `polyval` crates use a nightly-only `armv8` feature, although that's specifically because they're using the ARMv8 cryptography extensions.

I might suggest calling this feature `neon`. I guess `nightly` is ok too although doesn't really indicate what the feature does.

Reply:
Sure, I can rename it to be clearer. I originally went with nightly for familiarity, but now that I figured out how to not break `--all-features` with the crate-level features, a more targeted name makes sense.